akka-remote/src/main/mima-filters/2.5.8.backwards.excludes

```diff
@@ -0,0 +1,3 @@
+# #23732 Provide complete control over Netty SSL handler in Akka Remoting
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.remote.transport.netty.SSLSettings.this")
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.remote.transport.netty.NettyTransportSettings.this")
```

Comment:
Oh, one more thing, this should go in `2.5.11.backwards.excludes` now, sorry, didn't think of it the previous round.